src/deserialize.cpp

```diff
@@ -336,6 +387,87 @@ namespace sparrow_ipc
                     }
                 }
                 break;
+                case org::apache::arrow::flatbuf::Type::Timestamp:
```

Comment:
this swith start to be a bit big. We should create functions for each of the types. Let's do that in another PR